app/src/main/java/fr/neamar/kiss/loader/LoadDocsPojos.java

```diff
@@ -0,0 +1,112 @@
+package fr.neamar.kiss.loader;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.provider.MediaStore;
+
+import java.util.ArrayList;
+
+import fr.neamar.kiss.R;
+import fr.neamar.kiss.pojo.DocsPojo;
+
+import static android.provider.MediaStore.MediaColumns.DATA;
+
+public class LoadDocsPojos extends LoadPojos<DocsPojo> {
+
+    public LoadDocsPojos(Context context) {
+        super(context, "file://");
+    }
+
+    @Override
+    protected ArrayList<DocsPojo> doInBackground(Void... params) {
+        ArrayList<DocsPojo> docs = new ArrayList<>();
+
+        final String[] DOC_PROJECTION = {
+                MediaStore.Images.Media._ID,
+                MediaStore.Images.Media.DATA,
+                MediaStore.Files.FileColumns.MIME_TYPE,
+                MediaStore.Files.FileColumns.SIZE,
+                MediaStore.Images.Media.DATE_ADDED,
+                MediaStore.Files.FileColumns.TITLE
+        };
+
+        String[] selectionArgs = new String[]{".pdf", ".ppt", ".pptx", ".xlsx", ".xls", ".doc", ".docx", ".txt"};
```

Comment:
I am not sold on the idea of using JSON here. Could you talk some more about the benefits of using JSON in this case? How is it easier/faster to use a JSON parser to generate the `HashMap` of *file extensions → MIME types* and the `HashSet` of *MIME types*, then something else?
I do see the appeal of JSON generally but I'm curios what your motivations were in this case.

Also regarding the generated JSON array: Wouldn't it make more sense to have a single JSON object (instead of an array of object with only a single entry each)?
Like this:

    {
        "cwk " : "application/clarisworks",
        "mcw " : "application/macwriteii",
        "nb" : "application/mathematica",
        "nbp " : "application/mathematica",
        "doc" : "application/msword",
        …
    }